src/panels/config/js/trigger/numeric_state.js

```diff
@@ -24,7 +24,17 @@ export default class NumericStateTrigger extends Component {
   /* eslint-disable camelcase */
   render({ trigger, hass, localize }) {
     const { value_template, entity_id, below, above } = trigger;
+    let trgFor = trigger.for;
 
+    if (trgFor && (trgFor.hours || trgFor.minutes || trgFor.seconds)) {
```

Comment:
Can we fix the indentation here? looks off. Everything in the if should be indented